public/index.html

```diff
@@ -66,6 +66,17 @@
         alert('Sensorenregister is niet geschikt voor Internet Explorer.');
       }
     </script>
+
+    <script type="text/javascript">
```

Comment:
Hebben we ook een cookie banner nodig als we dit script plaatsen? Het lijkt er namelijk op dat er een cookie geplaatst/gelezen wordt.

Reply:
Ja. 

Lokaal kreeg ik een expliciete opt-in (gemeente managed laptop, Edge) voordat tracking ingeschakeld werd. Daarmee is de aanname dat deze implementatie AVG-proof is. Er moet nog wel nagevraagd worden naar welke privacy/cookie policy verwezen kan worden, maar amsterdam.nl en mijn.amsterdam.nl etc gebruiken PiwikPRO wel al. De betreffende cookie staat ook al op https://www.amsterdam.nl/privacy/cookies-site

Reply:
![image](https://user-images.githubusercontent.com/92784122/198006030-cbfc467e-e2cb-4e2c-8bb6-03fafae4c811.png)

De meldingen verschijnt inderdaad, maar de melding voldoet niet aan de huisstijl en er staat een groot piwik logo in. 
Dat lijkt me toch niet helemaal wenselijk. Daarnaast verwijst de link privacybeleid nergens naar toe.

Verder staan in de code nog verwijzingen naar het package `@datapunt/matomo-tracker-js`. Dat kan verwijderd worden.

Reply:
Overigens klopt de eerste zin van de melding ook niet.

Reply:
Dit is styling die het Data Analytics Platform kan oppakken. Ik zal dit als feedback naar hen sturen.